CONTRIBUTING.md

```diff
@@ -445,6 +447,24 @@ def test_something2(device):
         print("do something else.")
 
 
+
+@pytest.mark.with_device(["cpu", "gpu", tf.distribute.MirroredStrategy])
+def test_something3(device):
+    # the code here will run three times, once on gpu, once on cpu and once with 
+    # a mirror distributed strategy.
+    # device will be "cpu:0" or "gpu:0" or "gpu:1" or "gpu:2" or the strategy.
+    # with the MirroredStrategy, it's equivalent to:
+    # strategy = tf.distribute.MirroredStrategy(...)
+    # with strategy.scope():
+    #     test_function(strategy)
```

Comment:
```suggestion
    # the code here will run three times, once on gpu, once on cpu and once with 
    # a mirrored distribution strategy on two virtual gpu devices.
    # `device` will be either "cpu:0", "gpu:0", or  "gpu:0" / "gpu:1" respectively.
    # With the MirroredStrategy, it's equivalent to:
    # strategy = tf.distribute.MirroredStrategy(...)
    # with strategy.scope():
    #     test_function(strategy)
```
Slight re-wording. Per the `gpus_for_testing` function I'm not sure how we'd get a gpu:1 and gpu:2:
```
RMSprop/RMSprop/update_1/update_1/sub/x: (Const): /job:localhost/replica:0/task:0/device:GPU:1
RMSprop/RMSprop/update_0/Const: (Const): /job:localhost/replica:0/task:0/device:GPU:0
```

Reply:
Yeah, actually device is going to be cpu:0 or gpu:0 or the strategy. Let me reword that. Device can be the strategy too.